content/operate/rs/release-notes/rs-7-4-2-releases/rs-7-4-2-tba.md

```diff
@@ -0,0 +1,231 @@
+---
+Title: Redis Enterprise Software release notes 7.4.2-tba (June 2024)
+alwaysopen: false
+categories:
+- docs
+- operate
+- rs
+compatibleOSSVersion: Redis 7.2.0
+description: region_name field for new AWS S3 regions. Bug fixes.
+linkTitle: 7.4.2-tba (June 2024)
+weight: 67
+---
+
+This is a maintenance release for ​[​Redis Enterprise Software version 7.4.2](https://redis.com/redis-enterprise-software/download-center/software/).
+
+## Highlights
+
+This version offers:
+
+- `region_name` field for new AWS S3 regions
+
+- Bug fixes
+
+## New in this release
+
+### Enhancements
+
+- Added the `region_name` field in the REST API to import, export, and back up data in new AWS S3 regions.
+
```

Comment:
Why do we add release notes for the modules and not for "Redis database" itself?
(we have it only in GitHub I believe - e.g., https://github.com/redis/redis/releases/tag/7.2.5).